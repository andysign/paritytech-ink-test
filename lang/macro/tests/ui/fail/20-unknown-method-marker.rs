use ink_lang as ink;

#[ink::contract(version = "0.1.0")]
mod noop {
    #[ink(storage)]
    struct Noop {}

    impl Noop {
        #[ink(constructor)]
        fn new(&mut self) {}

        #[ink(message)]
        fn noop(&self) {}

        #[ink(unknown_marker)]
        fn has_unknown_marker(&self) {}

        #[ink(unknown_marker_2)]
        fn has_unknown_marker_too(&mut self) {}
    }
}

fn main() {}
