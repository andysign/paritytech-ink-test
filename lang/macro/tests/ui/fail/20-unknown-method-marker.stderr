error: unknown ink! marker
  --> $DIR/20-unknown-method-marker.rs:15:14
   |
15 |         #[ink(unknown_marker)]
   |              ^^^^^^^^^^^^^^^^
